### ISSUE TITLE
Add toggle to switch between open and closed issues/PRs on repos

### ISSUE THREAD

**Issue by rnystrom** [GitHawkApp/GitHawk#374]
Probably just put a unit at the top of the list?

**Comment by Sherlouk** [comment]
Relates to #287

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by ijm8710** [comment]
Is there any roadmap to add a open/closed toggle. #287 is closed itself

**Comment by ijm8710** [comment]
@rnystrom what were your thoughts on the value of tabbed or toggle options between open/close. As is I have to retype the word closed on the default open identifier. Saw in a chain you intentionally leaned primitive to start, but curious if you see value to polishing that or if you prefer to keep as is.

**Comment by rnystrom** [comment]
Ya I’d love at the very least a form or picker to make this easier. Right now it works but you have to know how to do all the filters.

It’d be cool to have like a filter icon/button in the bar. Tap it and we show a context menu form with all the options where you can select and enter all sorts of stuff (labels, status, etc) and maybe a link to GitHub search documentation for more advanced stuff.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Huddie** [comment]
#2679 has my design suggestion.


<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
I like it (and your handwriting). Not sure if that works with a standard `UISearchController` however. Can you take a stab at this in code and see what it'd look like?

If that doesn't work, what could be an alternative?

**Comment by Huddie** [comment]
I’ll try to code it up and think of alternatives as well. Not sure of a good alt. that doesn’t take up real estate 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
Yeah, it would then quickly take up real estate. I really like the current sketch, but that would also take away quite a size from the search bar horizontally... 🤷‍♂️

**Comment by Huddie** [comment]
That’s true but no more is:open necessary which opens up some text. I guess we’d have to see could be it takes up too much space 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Huddie** [comment]
@BasThomas turns out is not a SearchBarController :D lucky me. Just another cell, easy to manipulate. Check out #2687
